src/containers/Downloads/DownloadBar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Button from '../../components/Button';
+import Dropdown from '../../components/Dropdown';
+
+const DownloadBar = () => {
+  return (
+    <div className="downloads__bar">
+      <Button buttonStyle="secondary" text="Share" />
+      <div className="downloads__actions">
+        <div className="downloads__fieldset">
+          <label className="downloads__label">
+            Aggregate
+            <Dropdown
+              options={['Experiments', 'Samples']}
+              selectedOption={'Experiments'}
+            />
+          </label>
+          <label className="downloads__label">
```

Comment:
Do we need another className for `Transformation` here?

Reply:
Since it's using the same styles, we shouldn't need another className here.